hw/ip/dma/data/dma_testplan.hjson

```diff
@@ -295,7 +295,7 @@
               - Check if DMA generates an alert
               '''
         stage: V2S
-        tests: ["dma_illegal_transfer"]
+        tests: []
```

Comment:
My understanding is that it is *not* illegal to transfer between CTN and SYS, and we certainly don't prohibit this in the DMA controller presently, nor can I find any mention of such a requirement in the original specification document.
Maybe we should just remove this test point?

Reply:
I agree that we [currently don't specify restrictions for transfers between CTN and SYS](https://github.com/lowRISC/opentitan/blob/integrated_dev/hw/ip/dma/doc/theory_of_operation.md#security-considerations), and the way the `dma_illegal_transfer` testpoint is described, any direct transfers between CTN and SYS would be illegal. So I'm in favor of removing this testpoint without substitution.

@Razer6: Does this SGTY as well?

Reply:
Yes, there are no such restrictions. Removing that testpoint sounds good to me.

Reply:
Thx, removed.